dashboard/src/components/partials/ClearSilencedEntriesDialog/Dialog.js

```diff
@@ -138,18 +138,17 @@ class ClearSilencedEntriesDialog extends React.PureComponent {
       <Dialog fullWidth fullScreen={fullScreen} open={open} onClose={close}>
         <ListController
           items={silences}
+          initialSelectedKeys={silences.map(silence => silence.name)}
           getItemKey={node => node.name}
           renderEmptyState={this.renderEmpty}
           renderItem={this.renderListItem}
         >
           {({ children, selectedItems }) => (
             <Loader loading={submitting} passthrough>
               <DialogTitle>Clear Silencing Entries</DialogTitle>
-              <DialogContent style={{ paddingBottom: 8 }}>
-                <DialogContentText>
-                  Select all entries you would like to clear.
-                </DialogContentText>
-              </DialogContent>
+              <Typography style={{ paddingLeft: "24px" }}>
+                Select all entries you would like to clear.
+              </Typography>
```

Comment:
A margin below this text would improve the visual look of the modal

Reply:
<img width="622" alt="screen shot 2018-11-21 at 1 00 31 pm" src="https://user-images.githubusercontent.com/1707663/48868133-768e2980-ed8d-11e8-9158-9439bcfcf6fc.png">
Added a bit of a margin